website/layouts/sitemap.xml

```diff
@@ -0,0 +1,12 @@
+{{ printf "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"yes\"?>" | safeHTML }}
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+  {{- range .Data.Pages -}}{{ $currentVersion := index (split .Permalink "/" ) 1 }}{{ $currentVersionDir := $currentVersion | printf "/%s"}}{{ if and .Permalink (ne .Params.sitemap_exclude true) (eq $currentVersionDir site.Params.url_latest_version)}}
+  <url>
+    <loc>https://sidero.dev{{ .Permalink }}</loc>{{ if not .Lastmod.IsZero }}
+    <lastmod>{{ safeHTML (.Lastmod.Format "2006-01-02T15:04:05-07:00") }}</lastmod>{{ end }}{{ with .Sitemap.ChangeFreq }}
+    <changefreq>{{ . }}</changefreq>{{ end }}{{ if ge .Sitemap.Priority 0.0 }}
+    <priority>{{ .Sitemap.Priority }}</priority>{{ end }}{{ if .IsTranslated }}{{ range .Translations }}
+    <xhtml:link rel="alternate" hreflang="{{ .Language.Lang }}" href="{{ .Permalink }}"/>{{ end }}
+    <xhtml:link rel="alternate" hreflang="{{ .Language.Lang }}" href="{{ .Permalink }}"/>{{ end }}
```

Comment:
I _believe_ that's to handle ranging over the translations and then have the default language at the end. But that said, I'm not 100% sure either. It's the same in the upstream templae thought. We shouldn't hit it anyways though since we don't have translations.